### ISSUE TITLE
Detailed Logs in Admin UI

### ISSUE THREAD

**Issue by fahadpf** [socketio/socket.io-admin-ui#11]
Hello @darrachequesne 

Thank you for taking the time in improving our experience with [Socket.IO](/socketio/socket.io). I wanted to ask if there is any chance we can see the detailed logs (both access and error) of our server in Admin UI?

**Comment by darrachequesne** [comment]
By "detailed logs", you mean the logs of the HTTP server? Could you please explain what you have in mind?

**Comment by fahadpf** [comment]
Yes, it can also contain HTTP logs. Any errors that occurred in the Node process would also be quite helpful. This way, I would not have to switch back to my terminal to see what caused my server to restart unexpectedly.

Any connection errors should also be logged somewhere in the Admin UI.

**Comment by hamzahhourani** [comment]
which number of max clients can log in to admin ui?
